drivers/overlay/overlay.go

```diff
@@ -186,6 +189,15 @@ func parseOptions(options []string) (*overlayOptions, error) {
 			if err != nil {
 				return nil, err
 			}
+		case "overlay.imagestore":
+			// Additional read only image stores to use for lower paths
+			for _, store := range strings.Split(val, ",") {
+
+				if _, err := os.Stat(store); err != nil {
+					return nil, fmt.Errorf("overlay: Can't stat imageStore dir %s: %v", store, err)
+				}
+				o.imageStores = append(o.imageStores, store)
+			}
```

Comment:
I think these should be run through `path/filepath/Clean()`, and there should be a check for cases where `path/filepath/IsAbs()` returns `false`, because we may have to retrieve these values later, and there's no guarantee that we'll still be in the same working directory when we do.

May also want to verify that the `Stat_t` returned thinks that the path points to a directory.